cms/utils.py

```diff
@@ -11,3 +18,35 @@ def push_to_git(repo_path, index_prefix, es_host):
         remote.fetch()
         remote_master = remote.refs.master
         remote.push(remote_master.remote_head)
+
+
+def parse_repo_name(repo_url):
+    pr = urlparse(repo_url)
+    _, _, repo_name_dot_ext = pr.path.rpartition('/')
+    if any([
+            repo_name_dot_ext.endswith('.git'),
+            repo_name_dot_ext.endswith('.json')]):
+        repo_name, _, _ = repo_name_dot_ext.partition('.')
+        return repo_name
+    return repo_name_dot_ext
+
+
+def setup_workspace(
+        repo_path, index_prefix, es={'urls': [settings.ELASTICSEARCH_HOST]}):
```

Comment:
`settings.ELASTICSEARCH_HOST` here is not going to be overridable in tests. Rather assign it in the function?